stdlib/tkinter/ttk.pyi

```diff
@@ -961,9 +961,9 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
         master: tkinter.Misc | None = None,
         *,
         class_: str = ...,
-        columns: str | list[str] | list[int] | list[str | int] | tuple[str | int, ...] = ...,
+        columns: _TreeviewColumnId | list[str] | list[int] | list[str | int] | tuple[str | int, ...] = ...,
         cursor: tkinter._Cursor = ...,
-        displaycolumns: str | list[str] | tuple[str, ...] | list[int] | tuple[int, ...] = ...,
+        displaycolumns: _TreeviewColumnId | list[str] | tuple[str, ...] | list[int] | tuple[int, ...] = ...,
```

Comment:
I think we should delete `_TreeviewColumnId` and just use `int | str` everywhere. You can do it in this PR or in another PR or not at all, however you want :)